Products/zms/_objattrs.py

```diff
@@ -59,6 +59,8 @@ def getobjattrdefault(obj, obj_attr, lang):
         if default and datatype not in _globals.DT_TEXTS and isinstance(default, str):
           default = standard.dt_exec(obj,default)
         v = default
+    if obj.getSelf().id == 'e146':

```

Comment:
This snippet looks like a debug residual!?

Reply:
removed debug residual